src/analysis/normed_space/dual.lean

```diff
@@ -96,3 +109,81 @@ end
 end bidual_isometry
 
 end normed_space
+
+namespace inner_product_space
+open is_R_or_C continuous_linear_map
+
+variables (𝕜 : Type*)
+variables {E : Type*} [is_R_or_C 𝕜] [inner_product_space 𝕜 E]
+local notation `⟪`x`, `y`⟫` := @inner 𝕜 E _ x y
+local postfix `†`:90 := @is_R_or_C.conj 𝕜 _
+
+/--
+Given some x in an inner product space, we can define its dual as the continuous linear map
+λ y, ⟪x, y⟫.
+-/
+def to_dual (x : E) : normed_space.dual 𝕜 E :=
+linear_map.mk_continuous
+{ to_fun := λ y, ⟪x, y⟫,
+  map_add' := by simp only [inner_add_right, forall_const, eq_self_iff_true],
```

Comment:
I think that `:= λ _ _, inner_add_right` is a more readable proof, same for `map_smul'`.

Reply:
Indeed -- thanks!